CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fix
 
+- use anyio instead of aiofiles for async file I/O
```

Comment:
You can remove this change because it is done automatically by the CI

Reply:
done